### PR TITLE
kernel: fix confined mode

### PR DESCRIPTION
**Pull request by cgzones**

### CODE REVIEW

**Comment by pebenito** [comment]
I have a plan for removing deprecated interfaces. Please do not remove them. It also would have been better to group the changes by the type of change, e.g. one pull request with all of the deprecated interface removal.

**Comment by pebenito** [comment]
I see that most, if not all of these PRs are removing deprecated interfaces, please all revise PRs accordingly. I won't be repeating this message for all of them.

File: policy/modules/kernel/kernel.te [hunk 1]
```diff
@@ -276,6 +276,10 @@ dev_setattr_generic_blk_files(kernel_t)
 dev_setattr_generic_chr_files(kernel_t)
 dev_getattr_fs(kernel_t)
 dev_getattr_sysfs(kernel_t)
+dev_getattr_xserver_misc_dev(kernel_t)
+dev_setattr_xserver_misc_dev(kernel_t)
+dev_delete_xserver_misc_dev(kernel_t)
```
Comment (pebenito) [inline policy/modules/kernel/kernel.te]: What is deleting? A particular initramfs or something else?

**Comment by cgzones** [comment]
i currently do not use SELinux on this system anymore

**Pull request closed without merging**